analyzers/tests/SonarAnalyzer.UnitTest/Extensions/AssignmentExpressionSyntaxExtensionsTests.cs

```diff
@@ -27,6 +27,356 @@ namespace SonarAnalyzer.UnitTest.Extensions
     [TestClass]
     public class AssignmentExpressionSyntaxExtensionsTests
     {
+        [TestMethod]
+        public void AssignmentExpressionSyntaxExtensions_MapAssignmentArguments_TupleElementsAreExtracted()
+        {
+            var code = "(var x, var y) = (1, 2);";
+            var mapping = ParseAssignmentExpression(code).MapAssignmentArguments();
+            mapping.Should().BeEquivalentTo(new[]
+                {
+                    new
+                    {
+                        Left = new { Designation = new { Identifier = new { Text = "x" } } },
```

Comment:
This repeats many times and it's a lot of noise that makes it hard to understand the root purpose of the assertion
```suggestion
                        Left = CreateDesignation("x"),
```